website/react-magma-docs/src/pages/design/button-group.mdx

```diff
@@ -0,0 +1,167 @@
+---
+pageTitle: Button Group Design Guidelines
+title: Button Group
+order: 1
+---
+
+import { Link } from 'gatsby';
+import './design-guidelines-styles.css';
+import { DocsHeading } from '../../components/DocsHeading';
+import { LeadParagraph } from '../../components/LeadParagraph';
+
+<DocsHeading to="/api/button-group/" type="design">
+  Button Group
+</DocsHeading>
+<LeadParagraph>
+  The ButtonGroup component can be used to group related buttons.
+</LeadParagraph>
+
+![ ](../../images/button-group/button-group-intro.png)
+
+---
+
+## Options
+
+### Orientation
+
+A button group can be either horizontal or vertical in its orientation. By default, a button group is horizontal. Use vertical option when horizontal space is limited.
+
+<div class="two-col">
+  <div class="row">
+    <div class="col col-1">
+      <figure>
+        <img src="../../images/button-group/orientation.png" alt="" />
+      </figure>
+    </div>
+    <div class="col col-2">
+    </div>
+  </div>
+</div>
+
+### Size
+
+Button groups come in three different sizes: small, medium, and large. The medium size is the default and most frequently used option. Use the other sizes sparingly; they should be used to create a hierarchy of importance within the page.
+
+<div class="two-col">
+  <div class="row">
+    <div class="col col-1">
+      <figure>
+        <img src="../../images/button-group/size.png" alt="" />
+      </figure>
+    </div>
+    <div class="col col-2">
+    </div>
+  </div>
+</div>
+
+### No Space
+
+You can easily remove the space between buttons to create a single unit. When you join buttons like this together, there typically isn't a hierarchy within their relationship, so it is acceptable to use the same button style.
```

Comment:
Maybe it's obvious, but should we mention that no space is only for horizontal?

Reply:
I have updated the copy with your feedback.